docs/Project.toml

```
@@ -20,7 +20,7 @@ Trixi2Vtk = "bc1476a1-1ca6-4cc3-950b-c312b255ff95"
 TrixiBase = "9a0f1c46-06d5-4909-a5a3-ce25d3fa3284"
 
 [compat]
-CairoMakie = "0.12, 0.13, 0.14"
+CairoMakie = "0.12, 0.13, 0.14, 0.15"
```

```suggestion
CairoMakie = "0.15"
```
To force the newest version.